src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,146 @@
+html {
```

Comment:
why does files cards.scss contains styles related to html, container, etc, move them to another file